source/cellular_pkthandler.c

```diff
@@ -110,6 +110,13 @@ static CellularPktStatus_t _convertAndQueueRespPacket( CellularContext_t * pCont
                                              pContext->PktUsrDataLen );
         }
 
+        /* Reset the command type. Further response from cellular modem won't be
+         * regarded as AT command response. */
+        PlatformMutex_Lock( &pContext->PktRespMutex );
+        pContext->PktioAtCmdType = CELLULAR_AT_NO_COMMAND;
+        pContext->pRespPrefix = NULL;
+        PlatformMutex_Unlock( &pContext->PktRespMutex );
```

Comment:
Can users send next packet before we change the command type if we put thes part here after pktRespCB?